src/store/ConfigurationStore.ts

```diff
@@ -0,0 +1,35 @@
+import { defineStore } from "pinia";
+
+const useConfigurationStore = defineStore('configuration', () => {
+    // UNIPEPT
+    const unipeptApiUrl = 'https://api.unipept.ugent.be';
+
+    const parallelRequests = 5;
```

Comment:
In some earlier comment I suggested an environment variable for the amount of parallel requests. But we can also use the configuration store.

Reply:
Ah, that is actually better, I moved it to the `ConfigurationStore` indeed.